docs/guides/basics/install.md

```diff
@@ -1,4 +1,4 @@
-Floyd CLI is a Python-based command line tool to interact with FloydHub from your terminal.
+Floyd CLI (`floyd-cli`) is a Python-based command line tool to interact with FloydHub from your terminal.
 
 `floyd-cli` is available on [pypi](https://pypi.python.org/pypi/floyd-cli) and
 runs on both Python 2.7 and Python 3.5.
```

Comment:
I infrequently get questions on what OS is supported. May be add "on Linux, Mac OS or Windows"...

Reply:
Good suggestion. Added.